### ISSUE TITLE
Vovnet in CI having issues with downloading model

### ISSUE THREAD

**Issue by ajakovljevicTT** [tenstorrent/tt-xla#2390]
### Describe the bug

Vovnet models in CI are having issues with downloading from correcponsing `dropbox` links. Needs further investigation. 

https://github.com/tenstorrent/tt-xla/actions/runs/19807282505/job/56768721475#step:17:187

```
Running tests/runner/test_models.py::test_all_models_torch[vovnet/pytorch-vovnet39_th-single_device-full-inference] - pytorch_vovnet_vovnet39_th_cv_image_cls_torch_hub
Downloading: "https://dl.dropbox.com/s/1lnzsgnixd8gjra/vovnet39_torchvision.pth?dl=1" to /github/home/.cache/torch/hub/checkpoints/vovnet39_torchvision.pth
```

**Comment by kmabeeTT** [comment]
If it was recently deleted, we should find the original file and upload it to our s3 bucket then access via get_file(path) API like other models do already for these types of things.

**Comment by ddilbazTT** [comment]
@kmabeeTT @meenakshiramanathan1 We need to find a way to use the pre-trained weights. The untrained model gives incorrect labels when run through tt-inference-server: https://github.com/tenstorrent/tt-inference-server/pull/3898

**Comment by kmabeeTT** [comment]
Good catch... the change that went in on Feb 9 via https://github.com/tenstorrent/tt-forge-models/pull/435 to switch all models to use random weights is too aggressive... only a couple variants were affected and many were not (including vovnet27s that tt-inference-server serves) so I put up https://github.com/tenstorrent/tt-forge-models/pull/714 to partially revert that change (go back to using pretrained weights for those that work) and just merged it (solves the tt-inference-server issue when uplifting the last 5 months of changes).  This isssue should remain open though for the variants that don't have pretrained weights (those with TORCH_HUB source):

```

vovnet39_th | TORCH_HUB | Random (Dropbox 403 — tt-xla#2390)
vovnet57_th | TORCH_HUB | Random (Dropbox 403 — tt-xla#2390)
```
